README.md

```diff
@@ -1,8 +1,18 @@
 [![Build and Test](https://github.com/dafny-lang/dafny/workflows/Build%20and%20Test/badge.svg)](https://github.com/dafny-lang/dafny/actions?query=workflow%3A%22Build+and+Test%22) [![Gitter](https://badges.gitter.im/dafny-lang/community.svg)](https://gitter.im/dafny-lang/community?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
 
+Dafny is a **programming language teamed up with a verifier**. As you type in your program, the verifier constantly looks over your shoulders, flags any errors, shows you counterexamples, and congratulates you when your code matches your specifications. When you're done, Dafny **compiles your code to C#, Java, JavaScript and Go**, so it can integrate with your existing workflow.
```

Comment:
Typo: shoulders → shoulder
Also `Dafny **compiles your code to C#, Java, JavaScript and Go**` → `Dafny can **compile your code to C#, Java, JavaScript or Go**`

Reply:
Could we also say something like "with more languages to come (such as C++ and Python)"?

Reply:
I added (more to come) so that it does not distract from the sentence which should be simple.